### PR TITLE
Py38

### PR DESCRIPTION
**Pull request by eoinell**

### CODE REVIEW

**Comment by eoinell** [comment]
Can you guys have a double check this does the job? The whole architecture could definitely be simpler for sure though

**Comment by YagoDel** [comment]
I can't comment on the Trandor stuff. The ImageView stuff I hope gets superseded by #136

File: nplab/instrument/camera/camera_scaled_roi.py [hunk 1]
```diff
@@ -2,12 +2,9 @@
 """
 Subclass of Camera that has units for its axis. The GUI also provides crosshairs for defining ROIs
 """
-from __future__ import print_function
```
Comment (YagoDel) [inline nplab/instrument/camera/camera_scaled_roi.py:5]: I'm guessing we're now explicitly removing support for Python 2?

**Comment by YagoDel** [comment]
For imageview.py, the levelling of each individual image doesn't work. I've fixed this in #136 
If you're happy with the Renderers and the Trandor code, please merge this into master, and then we can merge #136 to fix imageview.py

**Comment by YagoDel** [comment]
I haven't looked at the merge conflicts yet, but I'm fairly certain they are from #135 and #120, which should take priority in the merge

**Pull request merged**